From edb1f9315df03f904b310efd2ba02d1e425a33f7 Mon Sep 17 00:00:00 2001
From: Philip Langdale <philipl@overt.org>
Date: Wed, 15 May 2019 11:09:51 -0700
Subject: [PATCH] vdpauinfo: Add names for the new 4:4:4 surface formats

Name mappings are required for vdpauinfo to report support is
available.

Signed-off-by: Philip Langdale <philipl@overt.org>
Reviewed-by: Andy Ritger <aritger@nvidia.com>
Reviewed-by: Manoj Bonda <mbonda@nvidia.com>
Signed-off-by: Aaron Plattner <aplattner@nvidia.com>
---
 vdpauinfo.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/vdpauinfo.cpp b/vdpauinfo.cpp
index d2b2688..25229fa 100644
--- a/vdpauinfo.cpp
+++ b/vdpauinfo.cpp
@@ -74,6 +74,8 @@ Desc ycbcr_types[] = {
 {"YUYV", VDP_YCBCR_FORMAT_YUYV},
 {"Y8U8V8A8", VDP_YCBCR_FORMAT_Y8U8V8A8},
 {"V8U8Y8A8", VDP_YCBCR_FORMAT_V8U8Y8A8},
+{"NV24", VDP_YCBCR_FORMAT_Y_UV_444},
+{"YV24", VDP_YCBCR_FORMAT_Y_U_V_444},
 };
 const size_t ycbcr_type_count = sizeof(ycbcr_types)/sizeof(Desc);
 
-- 
2.22.0

